doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/01_Apache_Configuration.md

```diff
@@ -4,7 +4,7 @@
 Make sure `Allowoverride All` is set for the `DocumentRoot`, which enables `.htaccess` support. 
 
 All the necessary rewrite rules, which are needed for Pimcore to work, are defined in the `.htaccess` of the install package, 
-see [https://github.com/pimcore/skeleton/blob/10.x/public/.htaccess](https://github.com/pimcore/skeleton/blob/10.x/public/.htaccess). 
+see [https://github.com/pimcore/skeleton/blob/10.1/public/.htaccess](https://github.com/pimcore/skeleton/blob/10.x/public/.htaccess). 
```

Comment:
```suggestion
see [https://github.com/pimcore/skeleton/blob/10.1/public/.htaccess](https://github.com/pimcore/skeleton/blob/10.1/public/.htaccess). 
```

Reply:
Oops 🙈

Reply:
I'm just thinking if we should really point to specific release branch as this tends to be removed with next minor release e.g. 10.2.

Reply:
@jdreesen please see 👆

Reply:
we can 🍒 pick [this ](https://github.com/pimcore/pimcore/pull/12619/files#diff-b9c0eecf9bf7df32f4c2b79c1d207fc75acb51575abc87972ef0139e9684e63b)on 10.4 branch

Reply:
> I'm just thinking if we should really point to specific release branch as this tends to be removed with next minor release e.g. 10.2.

Fair point, maybe we should just include the `.htaccess` in the docs for 10.4 as well? 
Older branches I won't fix, as they are unsupported anyway and just there for reference.

Reply:
Oh, well... Then maybe cherry-picking the mentioned commit may be the best solution.